polyhost/res/overlay_sources/doc_mark.py

```
@@ -0,0 +1,124 @@
+    "Document/SVG/ic_fluent_document_24_regular.svg")
+_c = pathlib.Path(__file__).resolve().parent / "_assets" / "fluent_document.svg"
+if not _c.exists():
+    _c.write_bytes(urllib.request.urlopen(urllib.request.Request(FL,
+        headers={"User-Agent":"polykybd"}), timeout=30).read())
+paths,_ = svgstr2paths(_c.read_text())
+rings=[]
+for p in paths:
+    for sub in p.continuous_subpaths():
+        pts=[(sub.point(t).real, sub.point(t).imag) for t in np.linspace(0,1,600)]
+        g=Polygon(pts)
+        if g.is_valid and g.area>0.5: rings.append(g)
```

Ruff (0.16.1)

[error] Multiple statements on one line (colon)

(E701)